sklearn/preprocessing/data.py

```diff
@@ -1065,6 +1077,12 @@ def robust_scale(X, axis=0, with_centering=True, with_scaling=True, copy=True):
         If True, scale the data to unit variance (or equivalently,
         unit standard deviation).
 
+    quantile_low : float
```

Comment:
Maybe, also say what are the constraints for the floats such that the param makes sense.